pkg/pgmodel/end_to_end_tests/main_test.go

```diff
@@ -40,15 +40,17 @@ func TestMain(m *testing.M) {
 		flag.Parse()
 		ctx := context.Background()
 		_ = log.Init("debug")
-		if !testing.Short() && *useDocker {
+		if !testing.Short() {
```

Comment:
Should we have a warning if we're `!Short && !useDocker`?

Reply:
In general, I'm not sure I agree with this change. From what I recall, when we originally added `useDocker` it was to be a strict subset of `!Short`, so you would add it if you wanted long tests but not the docker tests. Do we no longer think that's a needed case, or do we think that separating out into `end_to_end_tests` suffices?

Reply:
(I know at least me and @davidkohn88 don't always have docker running because of it's CPU implications, so being able to run only those tests that don't use docker seems like it might be useful still)

Reply:
I use it to run tests against a local db instance. That's very useful for debugging. In that case I wan't to run this test with use-docker=0. This was actually the use-case I created -use-docker for originally